src/parser.ts

```diff
@@ -37,6 +36,12 @@ export const EXECUTION_TYPES: Record<StatementType, ExecutionType> = {
   CREATE_FUNCTION: 'MODIFICATION',
   CREATE_INDEX: 'MODIFICATION',
   CREATE_PROCEDURE: 'MODIFICATION',
+  SHOW_DATABASES: 'LISTING',
+  SHOW_KEYS: 'LISTING',
+  SHOW_INDEX: 'LISTING',
+  SHOW_TABLE: 'LISTING', // for SHOW TABLE STATUS
+  SHOW_TABLES: 'LISTING',
+  SHOW_COLUMNS: 'LISTING',
```

Comment:
Why make these `LISTING` as opposed to `INFORMATION`? I get that `INFORMATION` has not been used before, so not sure if this would make a good usage.

Reply:
The reason is because INFORMATION was never used and wasn't sure if that was "intentional" or not. If you would like it to be INFORMATION, I'm totally cool with that.

Reply:
@MasterOdin , I made this change to have them as INFORMATION and have also added INFORMATION to the interface.

Reply:
Sorry, I was asking this question more as that's what @rathboma had asked for using `INFORMATION` in #67, and since you were implementing this at the request of @rathboma, was figuring you'd do that.

Having spent a bit of time over the last bit of week thinking about this, I think that using `LISTING` makes more sense since these queries do return data in that format and downstream clients probably won't care about the difference between `INFORMATION` and `LISTING` in these cases which makes it hard to reuse `INFORMATION` when we get to a query that does return something in a unique format.

Additionally, given that most (if not all) of these `SHOW` queries have generally equivalent queries you could run against the `INFORMATION_SCHEMA` tables and those would be `LISTING`, I think just keeping it all consistent makes sense.